### DISCUSSION TITLE
change the language of the distribution (ubuntu 20.04) ?

### DISCUSSION THREAD

**Discussion by Sid8899** [microsoft/WSL#7045] [Q&A]
### Windows Build Number

Microsoft Windows [version 10.0.19043.1023]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.4.72

### Distro Version

Ubuntu 20.04

### Repro Steps

sudo update-locale LANG=fr_FR.UTF8

### Expected Behavior

/

### Actual Behavior

*** update-locale: Error: invalid locale settings:  LANG=fr_FR.UTF8

### Diagnostic Logs

In official wsl docs(https://docs.microsoft.com/en-us/windows/wsl/faq#how-do-i-change-the-display-language-of-wsl-) it says we can change the language of wsl. We will have the exact same change as if in a real ubuntu installation ? 
I used this command to install the needed files : sudo apt-get install language-pack-XX language-pack-XX-base manpages 
is it the right one we should use on wsl 2  ?
Regards

**Comment by Biswa96** [comment]
I think it's `UTF-8`.

**Comment by Sid8899** [comment]
Hi, I tried with the "-" and it seems to be the same thing. Actually I'm able to change the language after doing sudo apt-get install language-pack-XX language-pack-XX-base manpages, otherwise refer to the questions I asked 

Thanks

**Reply by therealkenc** [reply]
> I tried with the "-" and it seems to be the same thing

It is.

> We will have the exact same change as if in a real ubuntu installation ?

Yes.

> [what is the] right one we should use on wsl 2

`sudo apt install language-pack-fr` will get you there, for sure. But just `sudo apt install locales` should be sufficient, unless you want translations, which you probably do (or at least might as well).

![image](https://user-images.githubusercontent.com/7049700/121110875-d66d4400-c7c2-11eb-91d0-e44c97bea4f7.png)

**Reply by Sid8899** [reply]
Hi thanks for your reply.  So for future people seeing this, excute that commands :

sudo apt-get install locales language-pack-fr language-pack-fr-base manpages

sudo update-locale LANG=fr_FR.UTF8

of course modify for your language,
